intro_to_r/intro_to_r.Rmd

```diff
@@ -66,7 +82,7 @@ boys to newborn girls, so he gathered the baptism records for children born in
 London for every year from 1629 to 1710.  We can take a look at the data by 
 typing its name into the console.
 
-```{r view-data, eval=FALSE}
+```{r view-data}
 arbuthnot
```

Comment:
Can data come already in the tbl_df format? E.g., can we build the package such that if students type data(present) the data they get is already a tbl_df? I am asking because I agree that the printouts are nicer, but I don't want to add unnecessary complication to the first lesson.